src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -378,50 +378,14 @@ def test_merged_sse_response(self, mock_read_write: pytest.MonkeyPatch) -> None:
         pd._preprocess_response(res_timestamps, res_outputs)
         assert res_outputs[1]["response"] == expected_response
 
-    def test_no_special_tokens(self, mock_read_write: pytest.MonkeyPatch) -> None:
```

Comment:
why was this test removed?

Reply:
It's pretty much duplicate since we have the same test in `test_tokenizer.py`.

Reply:
that works for me. 
thank you